EU4toV2/Source/V2World/V2Army.h

```diff
@@ -55,7 +55,7 @@ class V2Army // also Navy
 		std::string getName() const { return name; };
 		void getRegimentCounts(int counts[static_cast<int>(EU4::REGIMENTCATEGORY::num_reg_categories)]) const;
 		double getArmyRemainder(EU4::REGIMENTCATEGORY category) const { return armyRemainders[static_cast<int>(category)]; };
-		EU4::EU4Army getSourceArmy() const { return sourceArmy; };
+		EU4::EU4Army& getSourceArmy() { return sourceArmy; };
```

Comment:
Maybe rename this to getModifiableSourceArmy(). I feel like I've triggered the exact bug you're fixing before, and a better name might help future maintenance efforts avoid re-triggering it.